.gitignore

```diff
@@ -27,3 +27,4 @@ assets/
 container-ontology
 test/out
 .env
+out/DefaultBehaviorSpecialization.json
```

Comment:
Why is this being added to .gitignore? This seems very specific. Do you want to put the whole out directory in .gitignore instead?